.gitignore

```diff
@@ -41,4 +41,7 @@ terraform.tfstate.d/
 .terraform/
 .terraform.tfstate.lock.info
 .terraform.lock.hcl
-tokens
\ No newline at end of file
+tokens
+
+# Rails secrets
+/config/credentials/*.key
```

Comment:
Where will we store this key? I guess that's a wider question of 'where do we store secrets in a way visible to team members?'

Reply:
Zoho? 😛

Reply:
Seriously though, no idea. @cfq ?

Reply:
In our local ENV files?

Reply:
I have a secret key. I need to share it with the team. In other projects we've had a shared password manager for that sort of thing. How do we do it here?